db/migrate/20200726022620_add_stripe_customer_id_to_users.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class AddStripeCustomerIdToUsers < SolidusSupport::Migration[5.1]
+  def up
+    add_column Spree.user_class.table_name.to_sym, :stripe_customer_id, :string, unique: true
+
+    Spree::User.includes(bill_address: :country, ship_address: :country).find_each do |u|
```

Comment:
This part could be in a different migration. I believe with the custom user class situation, some users would want to deal with it themselves, and would be easier to skip a migration instead of comment this one.